nodebuilder/tests/p2p_test.go

```diff
@@ -61,19 +62,27 @@ Steps:
 8. Check LN is not allowed to dial with FN
 */
 func TestAddPeerToBlackList(t *testing.T) {
```

Comment:
Weird test, do you know what is the reason to have it in swamp? Looks like a unit test for connGater, which is external dependancy, so maybe it should be removed.

Reply:
Yeah I'm also unsure as to why that test is there. I'm happy to remove it - cc @Wondertan

Reply:
I am also happy to remove it. IIRC, it was added by @Bidon15 to ensure blacklisting works correctly once we exposed he feature for testground tests